docs/src/How To Work With Us.doc.js

```diff
@@ -0,0 +1,120 @@
+// @flow strict
+import React, { type Node } from 'react';
+import MainSection from './components/MainSection.js';
+
+const cards: Array<Node> = [];
+
+const card = (c) => cards.push(c);
+
+card(
+  <MainSection name="What goes into Gestalt?">
+    <MainSection.Subsection
+      description={`
+    The goal of Gestalt is to create a shared library of design best practices, React components, and documentation. Therefore, the best contenders for addition to Gestalt are designs or components that benefit multiple teams and are used regularly throughout our products.
+    `}
+    />
+  </MainSection>,
+);
+
+card(
+  <MainSection name="What is the process to request new additions or changes?">
+    <MainSection.Subsection
+      description={`
+What we build into Gestalt comes from the teams across Pinterest, so we’d love to pair with you early and often! The process starts in the design phase. Seeing early mocks, wireframes or even product roadmaps in our [partnership meetings](https://paper.dropbox.com/doc/How-to-Work-with-Us--BGVO2I8uo7TY0t~zzbc5PAY7Ag-KSFgsi7Me5kEqhC7sR2VA#:h2=Meetings-and-Events) to learn where your project may be going helps inform our team OKRs. The earlier we see the work, the better we’re able to plan and the higher the likelihood we’ll be able to help.
```

Comment:
this should be` [partnership meetings](#Meetings-and-Events)`

Reply:
Done